CK3ToEU4Tests/CK3WorldTests/TitleTests/TitleTests.cpp

```diff
@@ -101,7 +102,8 @@ TEST(CK3World_TitleTests, nameCanBeCleanedOfGUIJunk)
 
 	std::stringstream input2;
 	input2 << "key=\"k_grenada\"\n";
-	input2 << "name=\"\x15TOOLTIP:FAITH,catholic \x15L Catholic\x15!\x15! \x15ONCLICK:TITLE,733 \x15TOOLTIP:LANDED_TITLE,733 \x15L Lotharingia\x15!\x15!\x15!\"\n";
+	input2
+		 << "name=\"\x15TOOLTIP:FAITH,catholic \x15L Catholic\x15!\x15! \x15ONCLICK:TITLE,733 \x15TOOLTIP:LANDED_TITLE,733 \x15L Lotharingia\x15!\x15!\x15!\"\n";
```

Comment:
Probably unintended newline.

Reply:
That's clang doing clang things. Doesn't affect anything.